extensions/amp-analytics/0.1/requests.js

```diff
@@ -48,12 +50,17 @@ export class RequestHandler {
     /** @const {string} */
     this.baseUrl = dev().assert(request['baseUrl']);
 
-    /** @private @const {number} */
-    // TODO: to support intervalDelay that start timeout during construction.
+    /** @private {number} */
     this.maxDelay_ = Number(request['maxDelay']) || 0; //unit is sec
 
+    /** @private {Array<number>|number|undefined} */
+    this.batchInterval_ = request['batchInterval']; //unit is sec
```

Comment:
need to check if batchInterval is valid before I cast it to number. That's basically what `initBatchInterval_` is doing.

Reply:
Can we pass through a little helper method which tests for validity and always returns Array\<number\>|undefined? I think it would simplify matters to have the {number} case equivalent to an Array of size 1.

Reply:
batchInterval with value [1] or 1  are different now. Basically I need to figure the design for https://github.com/ampproject/amphtml/pull/13041#discussion_r164582819 first.